conf/rest/9.12.0/security_certificate.yaml

```diff
@@ -0,0 +1,40 @@
+
+name:             SecurityCert
+query:            api/security/certificates
+object:           security_certificate
+
+counters:
+  - ^^uuid
+  - ^name
+  - ^scope => scope
+  - ^type => type
+  - ^svm.name => svm
+  - ^public_certificate  => certificatePEM
+  - ^serial_number => serial_number
+  - expiry_time(timestamp) => expiry_time
+
+plugins:
+  - LabelAgent:
+      exclude_equals:
+        - scope `svm`
+      include_equals:
+#        - type `server` scope `cluster`   // TODO: To support multiple include with AND clause
```

Comment:
is this todo still needed?

Reply:
This functionality `anding with all include` hasn't been implemented yet. As it's not part of security dashboard, Will add it after this PR.